cmake/Modules/CMakeMacroAddFlags.cmake

```diff
@@ -7,8 +7,8 @@ MACRO(ADD_FLAGS parameter)
 	# Make a copy of the current arguments in ${parameter}
 	SET(new_parameter ${${parameter}})
 	
-        # fix arch-argument bug
-        STRING(REPLACE "-arch;" "-arch=" new_parameter "${new_parameter}")
```

Comment:
is the indentation change intentional?

Reply:
Yes. It was one level to much (same for the others)